src/test/test-integration.ts

```diff
@@ -34,20 +32,27 @@ describe("Integration test between client and service", function(){
     };
     client.onMessage(this._boardService.onClientConnection());
     this._clients.push(client);
-  }
+  };
 }
 
+describe("Integration tests with one client", function(){
+  var boardVM: BoardVM;
+  var server: ServerMock;
+  var boardService: BoardService;
+  var client: App;
+  beforeEach(function(){
+    client = new App(new BoardVM());
+    boardService = new BoardService();
+    server = new ServerMock(boardService);
+    server.addClient(client);
+  });
+
   it("Test with one client, case one: change, client tic, change server tic.", function(){
       // Arrange
         // Prepare variables
-      var boardVM: BoardVM = new BoardVM();
-      var client = new App(boardVM);
-      var noteVM = boardVM.createNote();
+      var noteVM = (<BoardVM>client.boardVM).createNote();
```

Comment:
Since `client.boardVM == boardVM` what do you think about to directly use `boardVM.createNote()` to avoid the cast?